Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
+
+# Prepare system packages
+RUN apt-get update
+
+# wget is need to get Elastix package
+# libgomp1 is required by elastix
+RUN apt-get -qq install wget libgomp1 -y
```

Comment:
I think you can get rid of the `wget` install. It was required in my docker to get the elastix packages from github, but since you are directly adding the binaries/libs below I do not think it is needed anymore.